experiments/osu-micro-benchmarks/mpi-only/ramble.yaml

```diff
@@ -34,7 +34,8 @@ ramble:
   software:
     packages:
       osu-micro-benchmarks:
-        pkg_spec: osu-micro-benchmarks
+        pkg_spec: osu-micro-benchmarks@comm_cali +mpi+openmp{modifier_spack_variant}
+        # pkg_spec: osu-micro-benchmarks
```

Comment:
@gracenansamba @pearce8 osu-micro-benchmarks does not support the mpi and openmp variants (mpi is always enabled). We also need to add the comm_cali branch to the package.py. It may be better to split the amg2023 and OSU changes into two separate PRs. The amg2023 PR works on ruby and produces the expected output.